db-backup/create-db-dump.sh

```diff
@@ -0,0 +1,9 @@
+#!/usr/bin/env bash
+set -euxo pipefail
+
+DB_URI=$(echo $VCAP_SERVICES | jq -r '.postgres[0].credentials.uri')
+
+pg_dump "${DB_URI}" --no-acl --no-owner | gzip | \
```

Comment:
I'd be interested to see if the custom format makes a big difference in terms of memory usage, size or speed of backup / restore, but that's something we can tweak later